src/test/java/org/javawebstack/orm/test/shared/knowledge/QueryKnowledgeBase.java

```diff
@@ -0,0 +1,36 @@
+package org.javawebstack.orm.test.shared.knowledge;
+
+import java.util.Arrays;
+import java.util.HashSet;
+
+public class QueryKnowledgeBase {
+
+    public static HashSet<String> sectionNames;
+    public static HashSet<Character> quoteCharacters;
+
+
+    static {
+        sectionNames = new HashSet<>(Arrays.asList(
```

Comment:
better word for this is `keywords` :)

Reply:
Yeah, that definitely better. Renamed to `TOP_LEVEL_KEYWORDS` as I don't mean to include keywords like `ON` in this list. Also made final.

Reply:
Now `TOP_LEVEL_SELECT_KEYWORDS`